### PR TITLE
Host metric support

### PR DESCRIPTION
**Pull request by jrb**
I think this addresses #75

### CODE REVIEW

**Comment by KierranM** [comment]
I'm not entirely sure why @duchatran's commit (712dfd3) is showing in the diff. Perhaps you need to do a rebase on master?

**Comment by jrb** [comment]
That was really strange, I'm not sure what happened. The branch has been corrected and replaced via a force push -- if you've made a local checkout of it, you'll want to delete and use the updated github branch.

**Comment by jrb** [comment]
This test failure looks like an issue with the upstream Chef repo. Is it possible to attempt another build without another commit?

**Comment by zl4bv** [comment]
Build restarted and now passing. Looks like there was a transient network failure.

File: test/integration/source-resource/serverspec/system_create_spec.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+require 'spec_helper'
+
+describe file('/etc/sumo.json/system.json') do
+  it { is_expected.to exist }
```
Comment (majormoses) [inline test/integration/source-resource/serverspec/system_create_spec.rb]: we should check its content?

Comment (jrb) [inline test/integration/source-resource/serverspec/system_create_spec.rb]: Probably, but that content based testing is probably a separate issue that needs to be addressed across most of the types in this cookbook.

Comment (majormoses) [inline test/integration/source-resource/serverspec/system_create_spec.rb]: fair enough, that being said I have found the only way to get that going is to start with new features

**Comment by majormoses** [comment]
@jrb can you rebase and fix conflicts?

**Comment by majormoses** [comment]
@jrb can you rebase and fix conflicts so I can (I am now a maintainer) get this over the line for you?

**Comment by jrb** [comment]
This is no longer something I'm in need of.

**Pull request closed without merging**